crates/agent/src/thread.rs

```diff
@@ -1546,9 +1546,9 @@ impl Thread {
                                             completion.queue_state =  QueueState::Started;
                                         }
                                         CompletionRequestStatus::Failed {
-                                            code, message
+                                            code, message, request_id
```

Comment:
Not directly related, but this is a new field we added to this variant in the new version of the `zed_llm_client`.